rosa_release_notes/whats-new-template.adoc

```diff
@@ -0,0 +1,6 @@
+// Text snippet included in the following modules:
```

Comment:
Why add this if it's not being used?

Reply:
This is for future usage, for other writers and PM and SME to potentially write their own what's new blurb. This is the template for that.